lib/games.rb

```diff
@@ -28,12 +29,12 @@ def event_summary(event)
     event.max_age.to_s.ljust(20) +
     event.min_age.to_s.ljust(20) +
     event.average_age.to_s.ljust(20) +
-    event.standard_deviation_age.to_s
+    event.standard_deviation.to_s
```

Comment:
Don't do this! The project specification says the method should be called `standard_deviation_age`. Instead of changing this file, you should change your method name.